filament/src/RenderPass.cpp

```diff
@@ -591,9 +593,16 @@ RenderPass::Command* RenderPass::generateCommandsImpl(RenderPass::CommandTypeFla
         // manual or hybrid instancing. Instanced stereo multiplies the number of instances by the
         // eye count.
         if (UTILS_UNLIKELY(hasStereo)) {
-            cmdColor.primitive.instanceCount =
-                    (soaInstanceInfo[i].count * stereoEyeCount) |
-                    PrimitiveInfo::USER_INSTANCE_MASK;
+            switch (stereoscopicType) {
```

Comment:
instead of passing `stereoscopicType`, just change the `hasStereo` to `hasInstancedStereo` and rename the `IS_STEREOSCOPIC` flag to `IS_INSTANCED_STEREOSCOPY`. This way we can remove the extra test and simplify this change a lot.

Reply:
Makes sense. Let me update it that way.